azurelinuxagent/common/osutil/arch.py

```diff
@@ -21,6 +21,8 @@
 from azurelinuxagent.common.osutil.default import DefaultOSUtil
 
 class ArchUtil(DefaultOSUtil):
+    jit_enabled = True
```

Comment:
DId you mean to make this static or instance? I think static is prefered, but in alpine you put it in __init__.  Please make it static everywhere.

Reply:
Changed everywhere.

---
In reply to: [192214413](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r192214413) [](ancestors = 192214413)

Reply:
Not sure I understand the rationale for making this static based on the usage. I would suggest you place this in the base class as an instance variable, and revert all the concrete class changes.

Reply:
since it never changes, there's only need to initialize it once.

---
In reply to: [194023160](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r194023160) [](ancestors = 194023160)

Reply:
It isn't the initialization that concerns me, it's the definition. It should be defined once in the base, and overridden only when necessary. I also think instance is preferable.

Reply:
Changed.

---
In reply to: [194177229](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r194177229) [](ancestors = 194177229)